css/home.css

```diff
@@ -0,0 +1,110 @@
+/* nav container  */
+
+/* .header {
+  height: 15vh;
+} */
+
+.header__nav {
+  background-color: transparent;
+  padding: 1rem;
+  overflow: hidden;
+}
+
+.header__nav--brand {
+  font-size: 1.8rem;
+  font-weight: var(--font-medium);
+  font-family: "Lobster", cursive;
+}
+
+.header__nav--search {
+  position: relative;
+}
+
+.fa-search {
+  position: absolute;
+  right: 0;
+  top: 20%;
+}
+
+.nav__search {
+  width: 400px;
+  padding: 0.5rem;
+  border: none;
+  border-radius: 1px;
+}
+
+.header__nav--menu > li {
+  list-style: none;
+}
+
+.btn {
+  background-color: var(--color-secondary);
+  font-size: 0.8rem;
+}
+
+/* main container  */
+
+.main {
+  width: 1280px;
```

Comment:
We are using px instead of rem because we want to have an absolute value of the main container instead of adjusting it to different sizes on different screens?Do let me know.

Reply:
Yes for this very reason I've given this a fixed width.